tests/zfs-tests/tests/functional/redacted_send/redacted_negative.ksh

```diff
@@ -58,14 +58,14 @@ log_mustnot zfs redact $sendfs@snap1 book
 log_mustnot zfs redact $sendfs#book1 book4 $clone1
 log_mustnot zfs redact $sendfs@snap1 book snap2 snap3
 log_mustnot zfs redact $sendfs@snap1 book @snap2 @snap3
-log_mustnot eval "zfs send --redact $sendfs#book $sendfs@snap >/dev/null"
+log_mustnot eval "zfs send --redact $sendfs#book $sendfs@snap >$TEST_BASE_DIR/devnull"
 
 # Redaction snapshots not a descendant of tosnap
 log_mustnot zfs redact $sendfs@snap2 book $sendfs@snap2
 log_must zfs redact $sendfs@snap2 book2 $clone1@snap $clone2@snap
 log_must eval "zfs send --redact book2 $sendfs@snap2 >$stream"
 log_must zfs redact $sendfs@snap2 book3 $clone1@snap $clone2@snap
-log_must eval "zfs send -i $sendfs@snap1 --redact book3 $sendfs@snap2 \
+log_must eval "zfs send -i $sendfs@snap1 --redact book3 $sendfs@snap2 |cat \
     >/dev/null"
```

Comment:
Shouldn't that be `>$TEST_BASE_DIR/devnull"` as well?

Reply:
Yes., good catch.